sdk/src/generated_sdk.rs

```diff
@@ -54136,7 +54137,7 @@ impl Client {
     /// `baseurl` is the base URL provided to the internal
     /// `reqwest::Client`, and should include a scheme and hostname,
     /// as well as port and a path stem if applicable.
-    pub fn new(baseurl: &str) -> Self {
+    pub fn new(baseurl: &str, inner: crate::LogCtx) -> Self {
```

Comment:
This seems like a pretty significant burr that we're imposing on all consumers.

Reply:
Agreed, adding an inner type is necessary to retain context with this approach.